ert_shared/storage/extraction.py

```diff
@@ -103,6 +103,13 @@ def create_response_records(ert, ensemble_name: str, observations: List[dict]):
     return records
 
 
+def _prepare_x_axis(x_axis: List[Union[int, float, str, pd.Timestamp]]) -> List[str]:
```

Comment:
would it be possible just writing a test for this function? I guess that should be quite simple since it is nicely separated and does not require any complex input or setup. Or have we decided to wait with the tests for the extraction logic for some reason?

Reply:
I can make a test